src/installer/pkg/sfx/Microsoft.NETCore.App/Directory.Build.props

```diff
@@ -35,6 +35,9 @@
     <_pastShimFiles Include="System.IO.Compression.Native.a" />
     <_pastShimFiles Include="System.IO.Compression.Native.dylib" />
     <_pastShimFiles Include="System.IO.Compression.Native.so" />
+    <_pastShimFiles Include="LdapProxy.Native.a" />
```

Comment:
Any changes  the `src/installer` should be reverted. Everything under installer is for netcoreapp. This file should not be included in netcoreapp.

Reply:
I was trying to get rid of:
```
D:\Sergey\git\runtime\.packages\microsoft.dotnet.sharedframework.sdk\6.0.0-beta.21179.7\targets\sharedfx.targets(273,5): error : The following files are missing entries in the templated manifest: [D:\Sergey\git\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
```
while running 
`build.cmd -subset host+packs -ci  -lc release -arch x64`

what is the right way to do this?

Reply:
It needs to be excluded somewhere earlier. It should not get this far.

Look for how System.IO.Ports.Native is done. The setup for this library should mirror it.

Reply:
Ok, will try, thank you!